libparsec/crates/types/src/id.rs

```diff
@@ -520,19 +520,35 @@ impl AsRef<str> for HumanHandle {
     }
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum HumanHandleParseError {
+    #[error("Email is missing")]
+    MissingEmail,
+    #[error("Invalid email address '{}'", .0)]
+    InvalidEmail(String),
```

Comment:
I'd rather avoid storing the bad email/label here as this prevent us from leaking sensitive data in logs & sentry reports.

Reply:
done